docs/theorymanual/source_sampling.rst

```diff
@@ -13,19 +13,20 @@ Theory
 ******
 
 Meshes can be used to represent particle source distributions for Monte Carlo
-radiation transport. On a mesh, source intensity is discretized spatially (into
-mesh volume elements) and by energy (into energy bins). In order to randomly
-sample these distributions to select particle birth parameters (position,
-energy, statistical weight) a discrete probability density function (PDF) must be
+radiation transport. On a mesh, source intensity is discretized spatially into
+voxel (mesh volume elements) or sub-voxel (intersection of a voxel and a cell)
+and by energy (into energy bins). In order to randomly sample these
+distributions to select particle birth parameters (position, energy, statistical
+weight, cell number) a discrete probability density function (PDF) must be
 created, which can be sampled with pseudo-random variates. It is convenient to
 create a single PDF to describe all of phase space; in other words, each bin
 within the PDF represents the probability that a particle is born in a
-particular energy group within a particular mesh volume element. 
+particular energy group within a particular voxel or sub-voxel.
```

Comment:
for unstructured mesh, 'voxel' is not the right term.  This sentence may need to be a little more complicated to cover all the cases.